akka-actor-typed-tests/src/test/java/akka/actor/typed/receptionist/ReceptionistApiTest.java

```diff
@@ -3,20 +3,95 @@
  */
 package akka.actor.typed.receptionist;
 
-
 import akka.actor.typed.ActorRef;
+import akka.actor.typed.ActorSystem;
+import akka.actor.typed.javadsl.AskPattern;
+import akka.actor.typed.javadsl.Behaviors;
+import akka.util.Timeout;
+
+import java.util.Set;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.TimeUnit;
 
 public class ReceptionistApiTest {
 
   public void compileOnlyApiTest() {
-    ActorRef<String> ref = null;
-    ActorRef<Receptionist.Registered<String>> listener = null;
-    ServiceKey<String> key = ServiceKey.create(String.class, "id");
-    Receptionist.Register register = new Receptionist.Register<>(key, ref, listener);
+    // some dummy prerequisites
+    final Timeout timeout = Timeout.apply(3, TimeUnit.SECONDS);
```

Comment:
AFAIR create exists and reads nicer in java

Reply:
Nope. Could be new `Timeout(Duration.create(3, TimeUnit.Seconds))` though. Not very important for this code though. It could just as well be `null`.

Reply:
oh doesnt, sux. So maybe `new Timeout()`

Reply:
yeah i guess, ok as is

Reply:
ah, theres an aux constructor taking int and TimeUnit